CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+1.  Update local master branch:
+
+        $ git checkout master
```

Comment:
Would be nice to indicate that this should be a fork and that an upstream have to be configured. A link would be enough.

Reply:
🆗